routes/shared.go

```diff
@@ -339,3 +339,12 @@ func (fes *APIServer) SendSeedBitClout(recipientPkBytes []byte, amountNanos uint
 	}
 	return hash, err
 }
+
+func BlockHashIn(slice []*lib.BlockHash, val *lib.BlockHash) bool {
+	for _, item := range slice {
+		if item.String() == val.String() {
```

Comment:
for comparing block hashes, we typically use reflect.DeepEqual. It may be a bit overkill, but we should be consistent.

Reply:
```suggestion
		if reflect.DeepEqual(item, val) {
```